<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Startfenster</class>
 <widget class="QDialog" name="Startfenster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>481</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QRadioButton" name="Client">
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>C&amp;lient</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="Server">
      <property name="text">
       <string>Ser&amp;ver</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbHost">
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="HostEingabe"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbPort">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="PortEingabe"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="VerbindenButton">
      <property name="text">
       <string>Verbinden</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="TrennenButton">
      <property name="text">
       <string>Trennen</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbNachrichten">
      <property name="text">
       <string>Nachrichten</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextBrowser" name="Verbindungsnachrichten"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="StartSpielfeld">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>500</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>40</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Schiffe versenken</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundlabel">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>-3</y>
     <width>651</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backgroundlabel</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>StartSpielfeld</zorder>
  <zorder>textLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
